### PR TITLE
get a human readable id for !humanid

### PR DESCRIPTION
**Pull request by balihoo-gens**

### CODE REVIEW

File: delbert/bot.py [hunk 1]
```diff
@@ -235,7 +235,7 @@ def is_plugin(obj):
                     env = self._get_plugin_env()
```
Comment (jsbronder) [inline delbert/bot.py]: Looks like some python3 compat fixes, mismatching the commit message.

File: test/test_humanid.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+class HumanIdTester(unittest.TestCase):
+    def setUp(self):
+        self._plugin = base.load_plugin('humanid.py', 'HumanId')
+        self._proto = base.TestProto([self._plugin])
+
+    @base.net_test
+    def test_query_real(self):
+        m = self._plugin.get_some()
+        self.assertEqual('A process for testing things', m)
```
Comment (jsbronder) [inline test/test_humanid.py]: This'll fail (every?) time.  Best you can probably do here is just make sure that ```m``` has non-zero length.

File: test/test_humanid.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        m = self._plugin.get_some()
+        self.assertEqual('test response', m)
+
+    @responses.activate
+    def test_msg(self):
+        ret = 'dr sugary booklet feat phat rule'
+        base.create_json_response('.*amazonaws.*', {'text': ret})
+
+        self._proto.privmsg('tester', base.TEST_CHANNEL, '!humanid')
+        print(self._proto.msgs)
```
Comment (jsbronder) [inline test/test_humanid.py]: remove

File: test/test_humanid.py (hunk 2 above)
Comment (balihoo-gens) [inline test/test_humanid.py]: i don't fucking know all your tests are shitty

**Comment by jsbronder** [comment]
rebase the above in and I'll pull.

**Pull request merged**